### ISSUE TITLE
Usage Button on Progress and Ranking page should be renamed to My Usage

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1192]
Button text should match destination "page" title, change Usage to My Usage to match destination page title.

**Comment by mosefer** [comment]
:+1:
